provision/minikube/keycloak/templates/keycloak.yaml

```diff
@@ -55,7 +55,7 @@ spec:
     vendor: mysql
 {{ end }}
 {{- if .Values.useAWSJDBCWrapper}}
-    url: jdbc:aws-wrapper:postgresql://{{ .Values.dbUrl }}:5432/keycloak
+    url: jdbc:aws-wrapper:postgresql://{{ .Values.dbUrl }}:5432/keycloak?sslmode=verify-full&sslrootcert=/opt/keycloak/certs/aurora.pem
```

Comment:
When we do it this way, I expect it would also show up in the docs eventually. In order for that to work, we would need to have a `podTemplate` configured, which is unsupported. 

This is then a block this issue, and it is soon do be deprecated - see [#40183](https://github.com/keycloak/keycloak/issues/40183)

There is an alternative way to configure this in https://github.com/keycloak/keycloak/issues/40184#issuecomment-3306911756 that is using `truststores` to pass in the name of the secret. 

With that change, I assume the `slrootcert=...` should no longer be necessary, and only `sslmode=verify-full` would be needed.

Reply:
Having said that, baking it in the image is not ideal, but for those (long lived) certificates, it might be an option. We are already adding the JDBC driver, and we do that in the same step.